packages/switch/README.md

```diff
@@ -95,42 +82,91 @@ focus to the content.
+Emphasized switches, which use the `empahasized` attribute/property are a
+secondary style for switches. The blue color provides a visual prominence
+that is optimal for forms, settings, etc. where the switches
 need to be noticed.
 
-```html-live
-<div style="display: flex; justify-content: space-between;">
-    <div style="display: flex; flex-direction: column;">
-        <h4 class="spectrum-Heading--subtitle1">Default</h4>
-        <sp-field-group selected="first" name="example" vertical>
-            <sp-switch emphasized value="off">Switch Off</sp-switch>
-            <sp-switch emphasized value="on" checked>Switch On</sp-switch>
-        </sp-field-group>
-    </div>
-
-    <div style="display: flex; flex-direction: column;">
-        <h4 class="spectrum-Heading--subtitle1">Disabled</h4>
-        <sp-field-group selected="first" name="example" vertical>
-            <sp-switch emphasized disabled value="off">Switch Off</sp-switch>
-            <sp-switch emphasized disabled value="on" checked>Switch On</sp-switch>
-        </sp-field-group>
-    </div>
-</div>
+```html
+<sp-field-group vertical>
+    <sp-switch emphasized>Emphasized</sp-switch>
+    <sp-switch emphasized checked>Emphasized and checked</sp-switch>
+</sp-field-group>
+```
+
+### States
+
+A switch can be disabled using the `disabled` property/attribute.
+
+```html demo
+<sp-field-group vertical>
+    <sp-switch invalid>Invalid</sp-switch>
+    <sp-switch disabled>Disabled</sp-switch>
+    <sp-switch disabled checked>Disabled and checked</sp-switch>
+</sp-field-group>
 ```
 
-### Handling events
+### Behaviors
+
+#### Handling events
 
 Event handlers for clicks and other user actions can be registered on an `<sp-switch>` similar to a standard `<input type="checkbox">` element.
 
 ```html
-<sp-switch id="switch-example" onclick="spAlert(this, '<sp-radio> clicked!')">
+<sp-switch id="switch-example" onclick="spAlert(this, '<sp-switch> clicked!')">
     Web component
 </sp-switch>
 ```
 
-## Accessibility
+### Accessibility
 
-Switch are accessible by default, rendered in HTML using the `<input type="checkbox">` element with the appropriate accessibility role, `switch`. When the Switch is `checked` or `invalid`, the appropriate ARIA state attribute will automatically be applied.
+Switch are rendered in HTML using the `<input type="checkbox">` element with the appropriate accessibility role, `switch`. When the Switch is `checked`, the appropriate ARIA state attribute will automatically be applied.
+
+#### Include a label
+
+A switch is required to have either a visible text label nested inside `<sp-switch>` itself.
+
+```html
+<sp-switch>Email notifications</sp-switch>
+```
+
+Standalone switches should be used in situations where the context is clear without an associated text label. For example, a switch located at the top of a panel next to the panel's title makes it clear that the switch will enable/disable the panel options.
```

Comment:
From [Design docs](https://spectrum.adobe.com/page/switch/#When-to-use-a-standalone-switch?) with added a11y recommendations.